internal/app/app.go

```diff
@@ -0,0 +1,89 @@
+package app
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+
+	"github.com/metal-toolbox/conditionorc/internal/model"
+	"github.com/spf13/viper"
+
```

Comment:
extra new line?

Reply:
fixed in [0335f0e](https://github.com/metal-toolbox/conditionorc/pull/1/commits/0335f0e3da4280dbdc079e734e9f72eb150d9abc)